qa/src/test/resources/credentials.properties

```diff
@@ -0,0 +1,22 @@
+## ---------------------------------------------------------------------------
```

Comment:
Could you just re-write the file and create it under the EPL license. Kapua itself cannot create content under the APL2.